src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -347,7 +347,7 @@ The following workflows contain a search space containing a set of possible neur
 
 The following workflows represent python templates that can be used to implement a generic machine learning task.
 
-*PyTorch_Task:* is a simple PyTorch task template.
+*Python_Task:* is a simple PyTorch task template.
```

Comment:
Why calling it `Python_Task` if it is a PyTorch task template? `PyTorch_Task` seems to be a more suitable task name for me.